source/03-components/mobile-menu/modules/_MobileMenu.es6.js

```diff
@@ -76,6 +76,14 @@ class MobileMenu extends OverlayMenu {
     if (blockClone.id) {
       blockClone.id = `${blockClone.id}-mobile`;
     }
+    const childrenWithId = [...blockClone.querySelectorAll('[id]')];
+    if (blockClone.matches('[id]')) {
+      childrenWithId.push(blockClone);
+    }
```

Comment:
I like how you're handling the children, but this seems like it duplicates what's right above it.
```
    if (blockClone.id) {
      blockClone.id = `${blockClone.id}-mobile`;
    }
```
Can lines 80-82 be removed?

Reply:
Good catch! Removed redundant code.